src/main/java/org/dependencytrack/persistence/jdbi/FindingDao.java

```diff
@@ -0,0 +1,435 @@
+               , "VULNERABILITY"."CVSSV2BASESCORE"
+               , "VULNERABILITY"."CVSSV3BASESCORE"
+               , "VULNERABILITY"."OWASPRRLIKELIHOODSCORE"
+               , "VULNERABILITY"."OWASPRRTECHNICALIMPACTSCORE"
+               , "VULNERABILITY"."OWASPRRBUSINESSIMPACTSCORE"
+               , "FINDINGATTRIBUTION"."ANALYZERIDENTITY"
+               , "VULNERABILITY"."PUBLISHED"
+               , "VULNERABILITY"."CWES"
+            <#if apiOrderByClause??>
+              ${apiOrderByClause}
+            </#if>
+            """)
+    @RegisterRowMapper(GroupedFindingRowMapper.class)
+    @AllowApiOrdering(by = {
+            @AllowApiOrdering.Column(name = "vulnerability.vulnId", queryName = "\"VULNERABILITY\".\"VULNID\""),
+            @AllowApiOrdering.Column(name = "vulnerability.title", queryName = "\"VULNERABILITY\".\"TITLE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.severity", queryName = "\"VULNERABILITY\".\"SEVERITY\""),
+            @AllowApiOrdering.Column(name = "vulnerability.cvssV3BaseScore", queryName = "\"VULNERABILITY\".\"CVSSV3BASESCORE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.cvssV2BaseScore", queryName = "\"VULNERABILITY\".\"CVSSV2BASESCORE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.published", queryName = "\"VULNERABILITY\".\"PUBLISHED\""),
+            @AllowApiOrdering.Column(name = "attribution.analyzerIdentity", queryName = "\"FINDINGATTRIBUTION\".\"ANALYZERIDENTITY\""),
+            @AllowApiOrdering.Column(name = "vulnerability.affectedProjectCount", queryName = "affectedProjectCount")
+    })
+    @AllowUnusedBindings
+    List<GroupedFinding> getGroupedFindings(@Define String queryFilter, @Define boolean activeFilter);
+
+    /**
+     * Returns a List of all Finding objects filtered by ACL and other optional filters. The resulting list is grouped by vulnerability.
+     *
+     * @param filters       determines the filters to apply on the list of Finding objects
+     * @param showInactive  determines if inactive projects should be included or not
+     * @return a List of Finding objects
+     */
+    default PaginatedResult getGroupedFindings(final AlpineRequest alpineRequest, final Map<String, String> filters, final boolean showInactive) {
+        StringBuilder queryFilter = new StringBuilder();
+        processFilters(filters, queryFilter);
+        final List<GroupedFinding> findings = withJdbiHandle(alpineRequest, handle ->
+                getGroupedFindings(String.valueOf(queryFilter), showInactive));
+        PaginatedResult result = new PaginatedResult();
+        result.setTotal(findings.size());
+        final List<GroupedFinding> findingsList = findings.subList(alpineRequest.getPagination().getOffset(),
+                Math.min(alpineRequest.getPagination().getOffset()
+                        + alpineRequest.getPagination().getLimit(), findings.size()));
+        result.setObjects(findingsList);
+        return result;
```

Comment:
Are we missing the aggregate filters, or are they implicitly covered with `processFilters`?

https://github.com/DependencyTrack/hyades-apiserver/blob/275227f1ac8307aa154c48b81bd9b2284c2d0ca8/src/main/java/org/dependencytrack/persistence/FindingsSearchQueryManager.java#L233-L242

https://github.com/DependencyTrack/hyades-apiserver/blob/275227f1ac8307aa154c48b81bd9b2284c2d0ca8/src/main/java/org/dependencytrack/persistence/FindingsSearchQueryManager.java#L293-L311

Can we make sure they still work? Unfortunately seems like there are no existing tests specifically for these filters.

Reply:
`processAggregatedDateRangeFilter` is not used anywhere.
And yes `processAggregateFilters` is part of grouped findings. I'll add it.